packages/polymath-js/src/contracts/STO.js

```diff
@@ -48,13 +49,33 @@ export default class STO extends Contract {
       startTime,
       endTime,
       cap,
-      rate,
+      rawRate,
       fundsRaised,
       investorCount,
       tokensSold,
       isPolyFundraise,
     ] = this._toArray(await this._methods.getSTODetails().call());
 
+    const factoryAddress = await this._methods.factory().call();
+
+    const GenericModuleFactory = new Contract._params.web3.eth.Contract(
+      IModuleFactoryArtifacts.abi,
+      factoryAddress
+    );
+
+    // NOTE @RafaelVidaurre: This is a hacky implementation to be backwards-compatible
+    // with 0.0.0 version of the Capped STO. This will be tackled properly by
+    // module version supporton the SDK
+    const version = (await GenericModuleFactory.methods
+      .getUpperSTVersionBounds()
+      .call()).join('.');
+
+    let rate = rawRate;
+
+    if (version === '0.0.0') {
+      rate = this._fromWei(rate).toNumber();
```

Comment:
Shouldn't this comparison be `!==`?

Reply:
Nope. version `0.0.0` is the one existing in `2.0.0` contracts

Reply:
What I mean is that in 2.0 the rate **doesn't** come as wei, so you should only make the conversion if the version is 2.1

Reply:
What, really?

Talk to you on Slack, I'm not sure this is the case